lib/subliminal/providers/podnapisi.py

```diff
@@ -82,7 +82,7 @@ class PodnapisiProvider(Provider):
     """Podnapisi Provider."""
     languages = ({Language('por', 'BR'), Language('srp', script='Latn')} |
                  {Language.fromalpha2(l) for l in language_converters['alpha2'].codes})
-    server_url = 'http://podnapisi.eu/subtitles/'
+    server_url = 'http://podnapisi.net/subtitles/'
```

Comment:
you must ask diaol to change that. Don't get why only change in medusa and not in subliminal repo